addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -660,15 +660,16 @@ def _import_fill_invoice_line_form(self, tree, invoice_line, qty_factor):
         tax_nodes = tree.findall('.//{*}Item/{*}ClassifiedTaxCategory/{*}Percent')
         if not tax_nodes:
             for elem in tree.findall('.//{*}TaxTotal'):
-                tax_nodes += elem.findall('.//{*}TaxSubtotal/{*}Percent')
+                tax_node = elem.findall('.//{*}TaxSubtotal//{*}Percent')
+                tax_nodes += tax_node
         return self._import_fill_invoice_line_taxes(tax_nodes, invoice_line, inv_line_vals, logs)
 
     def _correct_invoice_tax_amount(self, tree, invoice):
         """ The tax total may have been modified for rounding purpose, if so we should use the imported tax and not
          the computed one """
         # For each tax in our tax total, get the amount as well as the total in the xml.
         for elem in tree.findall('.//{*}TaxTotal/{*}TaxSubtotal'):
-            percentage = elem.find('.//{*}TaxCategory/{*}Percent')
+            percentage = elem.find('.//{*}TaxCategory//{*}Percent')
```

Comment:
useless diff

Reply:
No, it's not: Try to make my tests pass without it.

Reply:
@Levizar Done... no error :laughing:

Reply:
If you undo this one, you also have to undo the previous one:
![image](https://github.com/odoo/odoo/assets/52834046/2aaaf18a-773c-4874-82a0-8e2d30a9fa4d)

And when you do it, you get this:
![image](https://github.com/odoo/odoo/assets/52834046/fb8c4f65-7016-4883-becc-8e2961aa41c3)

Reply:
My bad, I understand what you mean.
You meant only the `+=` is useless diff previously but on tax category we always get the percent